packages/dds/tree/src/feature-libraries/forest-proxy/forestProxy.ts

```diff
@@ -0,0 +1,103 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { FieldKey } from "../../tree";
+import { IEditableForest, TreeNavigationResult, ITreeSubscriptionCursor } from "../../forest";
+
+export const proxySymbol = Symbol("forest-proxy");
+
+class TargetForest {
+	private readonly _cursor: ITreeSubscriptionCursor;
+
+	constructor(
+		public readonly forest: IEditableForest,
+		_cursor?: ITreeSubscriptionCursor,
```

Comment:
not sure why this has an underscore prefix

Reply:
It's the Fluid convention for handling conflicts between a field that is re-exposed as a property (below).

(Exposing as a readonly field would avoid the conflict.)